packages/ketcher-react/src/script/ui/state/floatingTools/index.ts

```diff
@@ -0,0 +1,63 @@
+import { Reducer } from 'react'
+import { Vec2 } from 'ketcher-core'
+
+export interface FloatingToolsState {
+  visible: boolean
+  handlePos: Vec2
```

Comment:
1) Could we define a more specific name for this state `handlePos`, from the usage - as I can see - it is more like: `toolBarPosition`? 

2) I would suggest making the interface stricter, like that: {x: number, y: number}, because it would be better to not allow to access 'Vec2' properties (that is related to renderer) from React level.

Reply:
`handlePos` is the center position of the rotation handle. 


I'd rename it to `rotateHandlePosition`.

Reply:
Second idea was about changing the interface, 
from the `Vec2` to `{x: number, y: number}`

why?
Because we don't need all internal logic from the Vec2 in our UI - correct?
I see only the plase where it is implemented:
`style={{ left: rotateHandlePosition.x, top: rotateHandlePosition.y }}` 

So we can do it, and in this case, late we can more easily refactor the implementation because we will have simpler interface.